pkg/s3utils/s3utils.go

```diff
@@ -93,7 +108,7 @@ func GetTimeStamp(getPreviousFolder bool) string {
 
 // GetS3Objects lists the objects in S3
 func GetS3Objects(clusterID string, nodeName string, getPreviousFolder bool) ([]Object, error) {
-	id, secret, endpoint, s3Region, disableSSLBool := GetAWSDetailsFromEnv()
+	id, secret, endpoint, s3Region, disableSSLBool, _, _, _ := GetAWSDetailsFromEnv()
```

Comment:
Create a separate function (say `GetS3SSEDetailsFromEnv`). Let us not change the existing function signature.

Reply:
done